docs/layers/line-layer.md

```diff
@@ -97,6 +97,35 @@ The stroke width of each object, in pixels.
 * If a number is provided, it is used as the stroke width for all objects.
 * If a function is provided, it is called on each object to retrieve its stroke width.
 
+It will be overridden by `getWidth` if this new accessor is provided.
+
+##### `getWidth` (Function|Number, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+* Default: `1`
+
+The stroke width of each object, in meters.
+
+* If a number is provided, it is used as the stroke width for all objects.
+* If a function is provided, it is called on each object to retrieve its stroke width.
+
+##### `widthScale` (number, optional)
+
+* Default: 1
+
+The scaling factor for the width of each line. If you set the property to `Math.pow(2, viewport.zoom - 12)` it will keep the width constant, corresponding to the current zoom level and the width of 1 pixel at zoom level 12. You can also limit the minimum size of the line with this property.
```

Comment:
The "If you set the property to..." sentence no longer applies with this update.